development-setup-wizard.mjs

```diff
@@ -0,0 +1,270 @@
+#!/usr/bin/env node --no-warnings
+
+/**
+ * Development setup wizard
+ *
+ * This script leads you through the steps required to get a working
+ * develompent version of camphoric up and running.  It should reflect what is
+ * in the doc/development.md file.
+ */
+
+import inquirer from 'inquirer';
+import chalk from 'chalk';
```

Comment:
Just a thought - the less a developer needs to do before running this script, the better it will streamline the setup procedure (which looks like the goal here). Writing it without any 3rd-party dependencies would help with that, and we'd be able to keep the security benefits of keeping dependencies isolated to the container.

Reply:
In principal I agree without, but chalk was already part of our dependency tree, so we aren't actually adding anything other than making it explicit that we use it as a core module.  I'm not sure if that affects how you feel about it.  I'm happy to take it out anyway, but you can see the yarn.lock diff isn't really anything. 

I think if we want to talk about removing lots of lib dependencies, inquirer and node-fetch are actually the big offenders.